backend/api/Utilities/PagedList.cs

```diff
@@ -37,7 +38,7 @@ int pageSize
             // but stops the runtime warning
             int totalCount = await source.CountAsync();
             List<T> items;
-            if (source.Expression.Type is IOrderedQueryable)
+            if (source.Expression.Type == typeof(IOrderedQueryable<MissionRun>))
                 items = await source.Skip((pageNumber - 1) * pageSize).Take(pageSize).ToListAsync();
             else
                 items = await source.OrderBy(x => 1).Skip((pageNumber - 1) * pageSize).Take(pageSize).ToListAsync();
```

Comment:
I think we can add a comment to explain why we are doing something else for MissionRuns